### PR TITLE
Handled the unknown and fallback element

### PR DESCRIPTION
**Pull request by IbrahimSulai**
This PR consist of the below,

1. Handled the unknown and fallback element
2. Handled the spacing and separator style

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/AdaptiveCards/pull/5536)

### CODE REVIEW

File: source/community/reactnative/src/models/model-factory.js [hunk 1]
```diff
@@ -21,7 +21,11 @@ export class ModelFactory {
             }
         } else {
             const elementModel = this.getElement(payload, parent);
-            return this.addCustomPropertyInModel(elementModel, payload)
+            if (elementModel) {
```
Comment (pragadeeshk) [inline source/community/reactnative/src/models/model-factory.js:24]: Please add element null check in requirements satisfied case too.

Comment (IbrahimSulai) [reply source/community/reactnative/src/models/model-factory.js:24]: Added element null check for requirements satisfied case also.

**Review by pragadeeshk — APPROVED** [review]
LGTM

**Review by vivekvijayakrishnan — APPROVED** [review]
LGTM

**Pull request merged**